projects/plugins/social/src/class-social-shares.php

```diff
@@ -66,13 +65,12 @@ public static function get_the_social_shares( $post_id = 0 ) {
 		$html = '<div class="jp_social_shares">';
 
 		if ( ! empty( $shares ) ) {
-			$html .= '<h3>' . __( 'Also on:', 'jetpack-social' ) . '</h5><ul>';
+			$html .= '<h5>' . __( 'Also on:', 'jetpack-social' ) . '</h5><ul>';
```

Comment:
I wonder if this wanted to be h5 or h3 😄

Reply:
As per this https://github.com/Automattic/jetpack/pull/36328#discussion_r1537572473, it's `h5`